defaults/main.yml

```diff
@@ -30,3 +30,7 @@ dr_clean_orphaned_disks: "True"
 
 # Indicate the default entities status report file name
 dr_report_file: "report.log"
+
+# Indicate the file name which is used to contain the data of the running VMs in the secondary setup before the failback
+# run again on the primary setup after the failback will be finished.
+dr_json_running_vms: "/usr/local/share/ansible/roles/ovirt-ansible-disaster-recovery/json_running_vms.json"
```

Comment:
Hmm, I think you should use relative path, as absolute path may differ in case of galaxy.